plugins/package-managers/gradle/src/main/kotlin/Gradle.kt

```diff
@@ -329,3 +316,14 @@ class Gradle(
         mavenSupport.close()
     }
 }
+
+private fun getGradleProperties(): Map<String, String> =
+    GRADLE_USER_HOME.resolve("gradle.properties")
+        .takeIf { it.isFile }
+        ?.inputStream()
+        ?.use { Properties().apply { load(it) } }
+        ?.mapNotNull { (key, value) ->
+            key.toString().takeUnless { it.startsWith("systemProp.") }?.let { it to value.toString() }
```

Comment:
Mind adding a commit to use this same line at

https://github.com/oss-review-toolkit/ort/blob/e773e2c1d8e73645c16ca5b6ab248a946a47005e/plugins/package-managers/gradle-inspector/src/main/kotlin/GradleInspector.kt#L278

as well? I believe the new code to be better as it does not blindly cast, but safely uses the `toString()` representation.

Reply:
> Mind adding a commit to use this same line at

I will make a separate PR for this.

_Do you think @sschuberth - it'd be an option to use a shared function between the two managers?_

Reply:
Sharing code certainly is an option (we're already partly doing so with the `gradle-model` module), but I was refraining from sharing more code as my hope was to remove the `gradle` module completely in favor of `gradle-inspector`. We should probably discuss this before investing more time to refactor code for sharing.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/10531.